package.json

```diff
@@ -15,9 +15,9 @@
     "lint": "sui-lint js && sui-lint sass",
     "test": "NODE_ENV=test npm run test:client && NODE_ENV=test npm run test:server",
```

Comment:
I'm not sure about this approach, what about every package manage its tests by itself? what about `sui-mono run npm test` in order to run all tests?

Reply:
Agree with sui-mono test 100%. You should create the task if carles cannot do it.
About executing separately I do not. The logic is already lint by repo and executing are supposed to be isolated so executing all at once has better performance, and no drawback I think.
You could still execute sui-mono test in a separate test.

Reply:
IMHO is a fact that we avoid that some test were broken due to what you are proposing. I mean, if we need to handle X parallel processes and 1 karma.conf each time that we create a new packages... I'm not sure if it is the best approach. What are the reasons that make you say that @julian-gernun?

On the other hand... what are we going to do with the test that is failing? @miduga @davidbarna @julian-gernun

Reply:
@carlesnunez it's DX I guess. To be able to work on one single package isolately